css/02-typography.css

```diff
@@ -372,15 +378,13 @@ main .card-title a:hover {
 }
 
 .card-time {
```

Comment:
We should also consider giving this color to `.card-date` as well.

Reply:
Good question. I think we can hold on this and see what it looks like in all contexts.